### DISCUSSION TITLE
How to save changes to HEAD in branch merge?

### DISCUSSION THREAD

**Discussion by codeyoulateralligator** [WinMerge/winmerge#692] [Q&A]
Hi,

I am trying to merge a branch into another branch. The behavior in Winmerge seems to have changed. I remember in the past there was a button like "Mark as resolved"? 

How should i use it now? If i make any changes to HEAD, click save then the changes are lost as soon as i reopen the file:

Winmerge:

![image](https://user-images.githubusercontent.com/61871309/112206944-35d5a200-8c1f-11eb-997b-a52943bc18d1.png)

See, i have added some random test line in the HEAD view (in Winmerge, basically manual conflict resolution). How do i now save this and mark as resolved so that it appears correctly in my merge commit?

I now click "Resolved" from here:
![image](https://user-images.githubusercontent.com/61871309/112207034-530a7080-8c1f-11eb-8952-47d33648b853.png)

But once i get to the merge commit i can see that the merge tags are still there. 

How to make manual changes to HEAD, save and mark as reolved?

**Comment by sdottaka** [comment]
Please tell me the contents of the Command Line displayed by selecting [Help]-> [Configuration] when WinMerge is started by selecting the "edit conflict" menu item.

I want to know if the /o option is set properly.

![image](https://user-images.githubusercontent.com/98126/112323358-0f823600-8cf5-11eb-9154-44ebe939e007.png)
